client/src/components/UpcomingTripsCard/UpcomingTripsCard.css

```diff
@@ -0,0 +1,60 @@
+.UpcomingTripsCard {
+    font-family: var(--default-font);
+    display: flex;
+    flex-direction: column;
+}
+
+.UpcomingTripsCard__header {
+    font-size: 2.25rem;
+    text-align: center;
+    color: var(--white);
+}
+
+.UpcomingTripsCard__info {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    padding: 0rem .1rem;
+    color: var(--white);
+}
+
+.UpcomingTripsCard__general-information {
+    font-size: 1.5rem;
+    font-weight: bold;
+    line-height: .3;
+    margin: 1.5rem 0 0 0;
+}
+
+ul {
+    list-style-type: none;
+    padding: 0;
+    font-family: var(--paragraph-font);
+    font-size: .9rem;
+    line-height: 1.75;
+}
```

Comment:
This needs to be namespaced under a class! Otherwise, it'll clash with other `ul` styling in other components